packages/@react-aria/actiongroup/src/useActionGroupItem.ts

```diff
@@ -36,6 +36,19 @@ export function useActionGroupItem<T>(props: ActionGroupItemProps, state: ListSt
     role: BUTTON_ROLES[selectionMode]
   };
 
+  // Cause rerender when focus key changes so tab index properly updates
+  let [, setFocusedKey] = useState(false);
```

Comment:
Now that useMultipleSelectionState doesn't trigger a rerender on `setFocusedKey` we have to subscribe any hooks that had previously relied on this behavior to changes in the focus key